src/generic/numberparsing.h

```diff
@@ -521,21 +1463,58 @@ really_inline bool parse_number(UNUSED const uint8_t *const src,
         // Ok, chances are good that we had an overflow!
         // this is almost never going to get called!!!
         // we start anew, going slowly!!!
```

Comment:
this comment should be updated?

Reply:
How so? 1. It is (hopefully) going to be rarely called. 2. It does involve starting anew.